src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/tests/Items.test.js

```diff
@@ -15,50 +15,40 @@ window.ResizeObserver = jest.fn(function() {
 });
 
 test('Render items', () => {
-    expect(render(<Items />)).toMatchSnapshot();
+    const {container} = render(<Items />);
+    expect(container).toMatchSnapshot();
 });
 
 test('Render items with children', () => {
-    expect(render(<Items><Button onClick={clickSpy}>Test</Button></Items>)).toMatchSnapshot();
+    const {container} = render(<Items><Button onClick={clickSpy}>Test</Button></Items>);
+    expect(container).toMatchSnapshot();
 });
 
 test('Resize div should call callback', () => {
```

Comment:
This test still needs to be migrated I need support on that one please because I have no idea how to check if a `div` has resized.

Reply:
i googled for a bit, but have not found an easy solution too 😕 would keep the test like it was before for the moment

Reply:
I reset the file back to its original state.